packages/types/config/build.d.ts

```diff
@@ -137,6 +137,7 @@ export interface NuxtOptionsBuild {
   analyze?: BundleAnalyzerPlugin.Options | boolean
   babel?: NuxtBabelOptions
   cache?: boolean
+  corejs?: 'auto' | 2 | 3
```

Comment:
We also support string '2' | '3' :)

Reply:
not according to documentation, see linked commit

Reply:
:/  @rchl docs PR was wrong at first place to mention number (reverted by 5e8933417fe809bec46043cce00ed2e73173fd8b to match 2.14) And now we do support both types.

Reply:
And you are sure that we want to extend it to string values also? I'd personally prefer more strict set of types but if you insist...

Reply:
Hmm i see. Since we didn't add types on 2.14.0 having types to force number is non-breaking.